lib/cosmos/tools/cmd_sequence/sequence_item.rb

```diff
@@ -280,6 +262,13 @@ def update_cmd_params(ignored_toggle = nil)
         end
       end
 
+      # If they passed in existing values override them here
+      if existing
+        existing.each do |param_name, param_value|
+          old_params[param_name] = param_value.to_s
```

Comment:
This is the essence of the fix. Use the parameter text directly instead of writing the underlying packet.